packages/app-tests/e2e/test-auth-flow.spec.ts

```diff
@@ -10,11 +12,11 @@ test.use({ storageState: { cookies: [], origins: [] } });
 */
 test.describe.configure({ mode: 'serial' });
 
-const username = process.env.E2E_TEST_AUTHENTICATE_USERNAME;
-const email = process.env.E2E_TEST_AUTHENTICATE_USER_EMAIL;
-const password = process.env.E2E_TEST_AUTHENTICATE_USER_PASSWORD;
+const username = 'Test User';
```

Comment:
Why are these hardcoded instead of using the environmental variables?

Reply:
Using single static credentials for all the tests is causing conflicts when some tests run in parallel. Given that the user fixture generates a random user, we may not require those environment variables, so these values are hard coded only for this specific test.

Reply:
Makes sense. I think I've seen the same thing in other repos as well.